jena-arq/src/main/java/org/apache/jena/sparql/util/Context.java

```diff
@@ -387,6 +389,11 @@ public void clear() {
         context.clear();
     }
 
+    /** Atomic compute. */
+    public <V> Object compute(Symbol key, BiFunction<Symbol, Object, V> remappingFunction) {
```

Comment:
This does mot seem to be used anywhere.

Reply:
It is used from the module `jena-fuseki-mod-geosparql` - class `SpatialIndexComputeService.scheduleTask`.
This method uses `Context.compute` to atomically check for a currently running indexing task if none exists creates a one.